Tests/CrossroadTests/AsyncAssertion.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+import XCTest
+
+func assertTrueAsynchronously(_ closure: @autoclosure () async -> Bool, message: String = "", file: StaticString = #file, line: UInt = #line) async {
+    let result = await closure()
+    if !result {
+        XCTFail(message, file: file, line: line)
+    }
```

Comment:
好みだと思いますが、私なら以下のように記述します。（False も同様です）

```suggestion
    XCTAssertTrue(result, message, file: file, line: line)
```

個人で似たような実装をしている（Crossroad を参考にさせていただきました :pray: ）ので、もし問題がありましたら教えていただけると嬉しいです！

Reply:
こっちの方がキレイですね。ありがとうございます